packages/shared/src/components/modals/LazyModalElement.tsx

```diff
@@ -1,12 +1,18 @@
-import React, { ReactElement } from 'react';
+import React, { ComponentType, ReactElement } from 'react';
```

Comment:
This file should not introduce the switch between apps, it will become a big pain down the line.